demisto_sdk/commands/content_graph/strict_objects/pack_meta_data.py

```diff
@@ -0,0 +1,65 @@
+from typing import List, Optional, Union
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.common import BaseStrictModel
+
+
+class StrictPackMetadata(BaseStrictModel):
+    name: str
+    display_name: Optional[str] = None
+    description: Optional[str] = None
+    created: Optional[str] = None
+    updated: Optional[str] = None
+    legacy: Optional[bool] = None
+    support: str
+    url: Optional[str] = None
+    email: Optional[str] = None
+    eula_link: Optional[str] = Field(None, alias="eulaLink")
+    author: str
+    author_image: str = Field("", alias="authorImage")
+    certification: Optional[str] = None
+    price: Optional[int] = None
+    hidden: Optional[bool] = None
+    server_min_version: Optional[str] = Field(alias="serverMinVersion")
+    current_version: Optional[str] = Field(alias="currentVersion")
+    version_info: str = Field("", alias="versionInfo")
+    commit: Optional[str] = None
+    downloads: Optional[int] = None
+    tags: List[str] = Field(default_factory=list)
+    categories: List[str] = Field(default_factory=list)
+    use_cases: List[str] = Field(default_factory=list, alias="useCases")
+    keywords: Optional[List[str]] = Field(default_factory=list)
+    search_rank: Optional[int] = Field(None, alias="searchRank")
+    excluded_dependencies: List[str] = Field(
+        default_factory=list, alias="excludedDependencies"
+    )
+    videos: Optional[List[str]] = Field(default_factory=list)
+    modules: Optional[List[str]] = Field(default_factory=list)
+    integrations: Optional[List[str]] = Field(default_factory=list)
+    hybrid: bool = Field(False, alias="hybrid")
+    default_data_source_id: Optional[str] = Field(None, alias="defaultDataSource")
+    default_data_source_name: Optional[str] = Field(None, exclude=True)
+    beta: Optional[bool] = None
+    dependencies: Optional[dict] = Field(default_factory=dict)
+    deprecated: Optional[bool] = None
+    marketplaces: Optional[List[MarketplaceVersions]] = None
+    github_user: Optional[Union[str, List[str]]] = Field(
+        alias="githubUser", default_factory=list
+    )
+    dev_email: Optional[Union[str, List[str]]] = Field(None, alias="devEmail")
+    displayed_images: Optional[List[str]] = Field(None, alias="displayedImages")
+    item_prefix: Optional[Union[str, List[str]]] = Field(None, alias="itemPrefix")
+    from_version: Optional[str] = Field(None, alias="fromVersion")
+    from_version_lower_case: Optional[str] = Field(None, alias="fromversion")
```

Comment:
let's remove this one and fix the single content item where it exists, instead of allowing an invalid key